assets/_scss/pages/_index.scss

```diff
@@ -106,11 +110,115 @@ body {
 
   .widget-header {
     margin-bottom: 1em;
-    text-align: center;
+  }
+}
+
+.slider-input {
+  -webkit-appearance: none;
+  width: 100%;
+  margin: 4.2px 0;
+  max-width: none;
 
-    h2 {
-      font-size: 1.5em;
+  &:focus {
+    outline: none;
+
+    &::-webkit-slider-runnable-track {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-ms-fill-lower {
+      background: lighten($color-gray-cool-light, 3);
     }
+    &::-ms-fill-upper {
+      background: lighten($color-gray-cool-light, 3);
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
+    width: 100%;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, cursor, height, width